src/modules/mavlink/mavlink_main.h

```diff
@@ -164,6 +164,11 @@ class Mavlink
 		MAVLINK_MODE_CONFIG
 	};
 
+	enum BROADCAST_MODE {
+		BROADCAST_MODE_OFF = 0,
+		BROADCAST_MODE_ON
+	};
```

Comment:
Because I'm pretty confident that this will end up being an enum with timeout values and stuff. On and off is certainly not enough to handle the myriad of auto-connect cases we already have now. The enum matches the MAVLink enum.